webapi/src/main/scala/org/knora/webapi/routing/v2/StandoffRouteV2.scala

```diff
@@ -69,8 +69,6 @@ final case class StandoffRouteV2()(
                 throw BadRequestException("part of HTTP multipart request has no name")
 
               case b: BodyPart => throw BadRequestException(s"multipart contains invalid name: ${b.name}")
-
-              case _ => throw BadRequestException("multipart request could not be handled")
```

Comment:
What is the story behind so many default cases being removed in this PR?

Reply:
Compiler warnings in case you have an exhaustive pattern match. The "default" case could only have been reached by matching on a null.

```scala
sealed trait A
case object X extends A
case object Y extends A

val foo: A = ???
foo match {
   case X => ???
   case Y => ???
   case _ => ??? // creates a compiler warning
   // case null => ??? // would be fine
 }}
 ```

Reply:
That was my guess after checking few cases in detail. Thanks for explanation.